runtime/shibuya/src/lib.rs

```diff
@@ -1324,10 +1373,119 @@ pub type Executive = frame_executive::Executive<
     Migrations,
 >;
 
+parameter_types! {
+    pub const BlockRewardName: &'static str = "BlockReward";
+}
 /// All migrations that will run on the next runtime upgrade.
 ///
 /// Once done, migrations should be removed from the tuple.
-pub type Migrations = ();
+pub type Migrations = (
+    pallet_inflation::PalletInflationInitConfig<Runtime, InitInflationParams>,
+    pallet_dapp_staking_v3::DAppStakingV3InitConfig<Runtime, InitDappStakingv3Params>,
+    frame_support::migrations::RemovePallet<
+        BlockRewardName,
+        <Runtime as frame_system::Config>::DbWeight,
+    >,
+    // This will handle new pallet storage version setting & it will put the new pallet into maintenance mode.
+    // But it's most important for testing with try-runtime.
+    pallet_dapp_staking_migration::DappStakingMigrationHandler<Runtime>,
+    // TODO: add migration to configure pallet inflation & dApp staking v3 init config
+);
+
+/// Used to initialize inflation parameters for the runtime.
+pub struct InitInflationParams;
+impl Get<pallet_inflation::InflationParameters> for InitInflationParams {
+    fn get() -> pallet_inflation::InflationParameters {
+        // TODO: set proper inflation parameters
+        pallet_inflation::InflationParameters {
+            max_inflation_rate: Perquintill::from_percent(7),
+            treasury_part: Perquintill::from_percent(5),
+            collators_part: Perquintill::from_percent(3),
+            dapps_part: Perquintill::from_percent(20),
+            base_stakers_part: Perquintill::from_percent(25),
+            adjustable_stakers_part: Perquintill::from_percent(35),
+            bonus_part: Perquintill::from_percent(12),
+            ideal_staking_rate: Perquintill::from_percent(50),
+        }
+    }
+}
+
+use frame_support::BoundedVec;
+use pallet_dapp_staking_v3::{EraNumber, TierParameters, TiersConfiguration};
```

Comment:
nitpick: Should be moved to top

Reply:
I've put it down here on purpose so it's easier to clean up later 🙂